### ISSUE TITLE
Publish 0.2

### ISSUE THREAD

**Issue by nox** [jmesmon/leak#5]
0.2 is the current version in the repository but isn't published on crates.io.

**Comment by codyps** [comment]
Yep, I've held off as I'm not sure about having `Leak::leak` return `&mut`. Might make sense to have a `LeekMut::leak_mut` or something. Or I could just change it later if it ends up making sense. Any thoughts?

**Comment by andrewhickman** [comment]
Until 0.2 is released the [documentation link on crates.io](http://codyps.com/docs/leak/x86_64-unknown-linux-gnu/stable/leak/trait.Leak.html) is out of date.
